src/base-element.js

```diff
@@ -901,22 +901,52 @@ export class BaseElement {
   }
 
   /**
-  * Runs the specified mutation on the element and ensures that measures
-  * and layouts performed for the affected elements.
-  *
-  * This method should be called whenever a significant mutations are done
-  * on the DOM that could affect layout of elements inside this subtree or
-  * its siblings. The top-most affected element should be specified as the
-  * first argument to this method and all the mutation work should be done
-  * in the mutator callback which is called in the "mutation" vsync phase.
-  *
-  * @param {function()} mutator
-  * @param {Element=} opt_element
-  * @return {!Promise}
-  */
+   * Runs the specified measure, which is called in the "measure" vsync phase.
+   * This is simply a proxy to the privileged vsync service.
+   *
+   * @param {function()} measurer
+   * @return {!Promise}
+   */
+  measureElement(measurer) {
+    return this.element.getResources().measureElement(measurer);
```

Comment:
Why not offer `opt_element` and plumb this through `this.runElement()` as well?

Reply:
> Why not offer opt_element 

There's no need for an element here, since no remeasures need to happen (remeasure needs a root element so we can optimize and only invalidate that element downwards).

> plumb this through this.runElement() as well?

For the moment, `#runElement` always revalidates. I didn't want to add another if statement. We can do it this way, though.